### PR TITLE
Add "falls through" comment explicitly.

### PR DESCRIPTION
**Pull request by interwq**
Fix warnings by -Wimplicit-fallthrough.

### CODE REVIEW

File: include/jemalloc/internal/sz.h [hunk 1]
```diff
@@ -61,7 +61,7 @@ sz_psz2ind(size_t psz) {
 		pszind_t lg_delta = (x < LG_SIZE_CLASS_GROUP + LG_PAGE + 1) ?
 		    LG_PAGE : x - LG_SIZE_CLASS_GROUP - 1;
 
-		size_t delta_inverse_mask = ZD(-1) << lg_delta;
+		size_t delta_inverse_mask = ZD(ZU(-1) << lg_delta);
```
Comment (davidtgoldblatt) [inline include/jemalloc/internal/sz.h]: I think we don't want a ZD at all, right? No sense in casting an unsigned to a signed back to an unsigned.

Comment (interwq) [reply include/jemalloc/internal/sz.h]: Yeah this doesn't really do anything. But I think we could get warnings from -Wsign-conversion w/o it?

Comment (davidtgoldblatt) [reply include/jemalloc/internal/sz.h]: Will we? It's unsigned shifted by unsigned, right? Or a ZU(lg_delta).

**Comment by interwq** [comment]
@davidtgoldblatt : removed the extra cast.

**Pull request merged**